packages/plugin-core/src/sourcemaps.ts

```diff
@@ -33,19 +35,21 @@ export async function uploadSourcemaps(sourcemapData: SourcemapInfo[], hbOptions
     const errorsStr = rejected.map(p => p.reason).join('\n')
     throw new Error(`Failed to upload ${rejected.length} sourcemap file(s) to Honeybadger\n${errorsStr}`)
   }
-  
+
+  console.info(`Note: For the error to be matched with a source map, the revisions must match. You can learn how to configure honeybadger.js to include the revision here: ${DOCS_VERSIONING_URL}`)
```

Comment:
It looks like we're checking an `hbOptions.silent` config option for the success message logging. I think we should also respect that here?

Reply:
I considered that too, but the `silent` option is like a debug/verbose flag. My understanding was that we would print this message at all times, not only in debug mode. I don't have a strong opinion though, we can check for silent here as well. Let me know if you want me to go for it.

Reply:
Oh, you are right, I don't know how I confused that bit 😓 .
In that case, I'll implement your suggestion and check for the flag.